src/D2L.Security.OAuth2/D2L.Security.OAuth2.csproj

```diff
@@ -178,7 +178,7 @@
     <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
-    <Analyzer Include="..\..\packages\D2L.CodeStyle.Analyzers.0.55.0\lib\net461\D2L.CodeStyle.Analyzers.dll" />
+    <Analyzer Include="..\..\packages\D2L.CodeStyle.Analyzers.0.74.0\lib\net47\D2L.CodeStyle.Analyzers.dll" />

```

Comment:
Seems like a discrepancy between this and the `targetFramework` in pacakges.config